frontend/package.json

```diff
@@ -33,5 +33,9 @@
     "postcss": "^8.4.32",
     "tailwindcss": "^3.4.0",
     "vite": "^5.0.8"
-  }
+  },
+  "main": "postcss.config.js",
+  "author": "",
+  "license": "ISC",
+  "description": ""
```

Comment:
@Kratik1093 why did you aded these lines here?

Reply:
to make this page run on my device i need to made these change otherwise the page was not starting on my device

Reply:
@Kratik1093 then please remove this while pushing the PR and use it for local only

Reply:
ok will generate another pr removing this part

Reply:
@Kratik1093 the above lines need to be removed to merge this Pr. Please remove it asap.